filescan_sandbox_result.py

```diff
@@ -96,103 +115,170 @@ def process_allSignalGroups(result_section, all_signal_groups):
 
 
 def process_iocs(result_section, iocs):
-    def process_ioc(key, target_tag):
-        if key in iocs:
-            for element in iocs[key]:
-                if 'data' in element:
-                    return (target_tag, element['data'])
-        return None
+    origin_type_map = {
+        "INPUT_FILE": "static",
+        "INPUT_CONTEXT": "static",
+        "EXTRACTED_FILE": "static",
+        "DOWNLOADED_FILE": "static",
+        "URL_REDIRECT": "static",
+        "CONTENT_PARSE": "static",
+        "EMAIL_BODY": "static",
+        "EXTERNAL_PARSER": "static",
+        "JAVA_DECOMPILATION": "static",
+        "DOTNET_DECOMPILATION": "static",
+        "APK_DECODING": "static",
+        "AUTOIT_DECOMPILATION": "static",
+        "IMAGE_OCR": "static",
+        "IMAGE_QR_CODE": "static",
+        "OLE_PARSE": "static",
+        "PCODE_PARSE": "static",
+        "DOMAIN_RESOLVE": "static",
+        "LNK_PARSE": "static",
+        "XOR_DECODING": "static",
+        "INTERNAL": "static",
+        "URL_RENDER": "dynamic",
```

Comment:
I think we are missing:
`"PE_UNPACKING": "dynamic",`

Reply:
Uh, you're right! Nice catch, thanks!